src/core/effects/qgsgloweffect.cpp

```diff
@@ -176,7 +176,14 @@ void QgsGlowEffect::readProperties( const QgsStringMap &props )
 
   //attempt to create color ramp from props
   delete mRamp;
-  mRamp = QgsGradientColorRamp::create( props );
+  if ( props.contains( QStringLiteral( "schemeName" ) ) )
```

Comment:
Yeah. Otherwise it'll get very messy if additional gradient type ramps are added.